make_site.py

```diff
@@ -173,6 +173,18 @@ class HundredTheorem:
     links: Optional[Mapping[str, str]] = None
     note: Optional[str] = None
 
+@dataclass
+class ThousandPlusTheorem:
+    # Wikidata identifier, e.g. "Q1008566"
+    number: str
```

Comment:
the "number" occasionally ends in an `X`/`A`/`B`, which is probably included in this field. Maybe add a comment that this does include these trailing letters?